src/spikeinterface/qualitymetrics/pca_metrics.py

```diff
@@ -53,7 +53,10 @@
         n_components=10,
         radius_um=100,
         peak_sign='neg'
-    )
+    ),
+    silhouette=dict(
+        method='simplified'
```

Comment:
```suggestion
        method=('simplified',)
```
Since the code can take a tuple as well, I would keep the tuple by default.

The `if "simplified" in qm_params["silhouette"]["method"` also makes more sense with tuple (it should work with strings too, but in that case a `==` would be easier tot read)

Reply:
no I think that's ok, but let's stick to the `tuple` in the params, because a tuple is actually accepted.

Reply:
Already made `tuple` 👍. This was in addition. But as long as you're okay with that then I'll leave the `if` as is.